iso4217parse/__init__.py

```diff
@@ -207,6 +208,12 @@ def by_symbol_match(value, country_code=None):
             if group == 'name':
                 res = [_data()['name'][s]]
             if res:
+                if country_code:
+                    res = [
```

Comment:
I think, if `res` is empty after the `country_code` filtering, the search should continue. Please put the `if country_code:` branching outside and before the `if res:` branch.